server/routes/user.js

```diff
@@ -20,8 +26,8 @@ router.get('/', async (req, res) => {
     }
     res.json(user);
   } catch (error) {
-    console.log(error);
     res.sendStatus(500);
+    next(error);
```

Comment:
@LGG233 I also learned Express has a built-in default error handling middleware which can be triggered (and thus result in some console logging of the error) by calling `next(errorObject)`. There's some other small benefits to doing this instead of explicitly console logging, so I'll lean this direction (use `next` instead of `console.log`) in the future when wanting to log server errors.